File: Front/ComparaImovel-App/src/app/models/Cliente.ts
import { Imovel } from './Imovel';
import { RedeSocial } from './RedeSocial';

export interface Cliente {
    id: number;
    nome: string;
    ocupacao: string;
    imagemURL: string;
    telefone: string;
    email: string;
    redesSociais: RedeSocial[];
    clientesImoveis: Imovel[];

}

File: Back/src/ProEventos.Domain/Cliente.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ProEventos.Domain
{
    public class Cliente
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public string Ocupacao { get; set; }
        public string ImagemURL { get; set; }
        public string Telefone { get; set; }
        public string Email { get; set; }
        public IEnumerable<RedeSocial> RedesSociais { get; set; }
        public IEnumerable<ClienteImovel> ClientesImoveis { get; set; }

    }
}
